File: Python Contest/dwite-200410-p2.py
import math

input = "00:00"

h = input[0:2]
h = int(h)

m = input[3:5]

XM = " AM"

def calcTime():
	if 12 <= h <= 23:
		XM = " PM"
		h = h - 12

	if h == 0:
		h = 12

	h = str(h)

	print(h + ":" + m + XM)

calcTime()

File: Python Contest/dwite-200410-p2.js
input = "00:00";

h = input.substring(0, 2);
h = parseInt(h);

m = input.substring(3, 5);

XM = " AM";


if (12 <= h && h <= 23) {
	XM = " PM";
	h = h - 12;
}
else if (h == 0) {
	h = 12;
}

console.log(h + ":" + m + XM);